src/SlippiGame.ts

```diff
@@ -194,6 +207,43 @@ export class SlippiGame {
     return stats;
   }
 
+  public getStadiumStats(): StadiumStatsType | null {
+    this._process();
+
+    const settings = this.parser.getSettings();
+    if (settings === null) {
+      return null;
+    }
+
+    const latestFrame = this.parser.getLatestFrame();
+    const players = latestFrame?.players;
+
+    if (!players) {
+      return null;
+    }
+
+    this.stadiumStatsComputer.process();
+
+    let sandbag = null;
+    for (let i = 0; i < settings.players.length; i++) {
+      sandbag = players[i]?.post.internalCharacterId === 32 ? players[i]?.post : null;
+    }
+
```

Comment:
```suggestion
  switch (settings.gameMode) {
    case GameMode.TARGET_TEST:
      // Return TargetTestResultType
      break;
    case GameMode.HOME_RUN_CONTEST:
      // Return HomeRunContestResultType
      break;
    default:
      return null;
  }
```
Can we use a switch-case here?

Reply:
updated to use a switch statement - please let me know if there is a cleaner way to filter to the `Sandbag` character